server/reports/requisition/2_5_0/src/template.html

```diff
@@ -62,13 +65,11 @@
         <td class="item_name" style="width: 25%">{{ line.item.name }}</td>
         <td class="quantity" style="width: 8%">{{ line.itemStats.availableStockOnHand }}</td>
         <td class="pack" style="width: 10%">{{ line.itemStats.averageMonthlyConsumption | round(precision=2) }}</td>
-        <td class="stock_to_order" style="width: 8%">
-          {{ data.requisition.maxMonthsOfStock * line.itemStats.averageMonthlyConsumption }}
-        </td>
+        <td class="mos" style="width: 8%">{{ line.itemStats.monthsOfStockOnHand }}</td>
```

Comment:
Nice- can we just round so it is consistent w AMC column 🙏 

![Screenshot 2025-01-22 at 4 28 04 PM](https://github.com/user-attachments/assets/3e03925d-4187-408f-8de4-91755d8efa0d)

And maybe suggested quan should be right aligned hehe